dask/dataframe/io/parquet.py

```diff
@@ -948,27 +948,31 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
             if path.open != fastparquet.util.default_open:
                 assert (re.match('.*://', path.fn)), \
                        ("ParquetFile: Path must contain protocol" +
-                        " (e.g., s3://...) when using other than the default" + 
+                        " (e.g., s3://...) when using other than the default" +
                         " LocalFileSystem. Path given: " + path.fn)
 
-            assert (path.file_scheme == 'hive'), "Only 'hive' is currently supported as the ParquetFile's file scheme"
             is_ParquetFile = True
     except ImportError:
             pass
 
     if is_ParquetFile:
         read = get_engine('fastparquet')['read']
+        if path.file_scheme == 'hive':
+            urlpath = path.fn.split('_metadata')[0]
```

Comment:
I think the original logic was fine here: if the last part is `_metadata`, remove it. This can happen, for example, with the drill file-scheme.

Reply:
OK I was making assumptions about the way it works.
I think now the logic is more robust. Also works if in the middle of the path there is a dir name '_metadata'. Don't know why someone would do that. but we never know...